apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -171,11 +175,38 @@ const PythonlabView: React.FunctionComponent<
     dispatch(setWidgetViewShowCode(false));
   });
 
+  // Given a question for the AITutor2, return the full question to ask, which means
+  // appending all the relevant context.
+  const getAiTutor2FullQuestionFromQuestion = (
+    question: string,
+    questionType: 'hint' | 'user'
```

Comment:
Is `questionType` used here?

Reply:
Not currently, so I'll remove.  I had been thinking we might give different context depending on the question type, such as not including validation results in the freeform chat.  But we can figure that out later as we learn more.